packages/expo-file-system/ios/EXFileSystem/EXFileSystem.m

```diff
@@ -703,6 +730,61 @@ - (NSDictionary *)encodingMap
 
 #pragma mark - Internal methods
 
+- (EXFileSystemUploadType)_importUploadType:(NSNumber * _Nullable)type
```

Comment:
```suggestion
- (EXFileSystemUploadType)getUploadTypeFrom:(NSNumber * _Nullable)type
```
Semantically what you're doing here is just getting import type from some integer value with fallback when value is not in available ENUM values.